requirements.txt

```diff
@@ -3,6 +3,7 @@ torch>=1.10.0
 torchvision>=0.11.0
 pytorch-lightning==1.7.1
 torchmetrics==0.10.0
+torch_geometric>=2.1.0
```

Comment:
I think this is the wrong package for pip, I'm showing it as "torch-geometric". We will probably also need to add "torch-scatter" and "torch-sparse" at minimum to get the tests running in the CI environment